services/line-stat-getter.js

```diff
@@ -108,7 +136,7 @@ function LineStatGetter(model, params, opts) {
       records = _.sortBy(records, 'label');
       return _.map(records, function (record) {
         return {
-          label: moment(record.label).format(getFormat()),
+          label: moment(record.label, record.format).format(getFormat()),
```

Comment:
Maybe I miss something, but why don't you call the right format while formatting the label?
Once again why do you need to pass the format (which is always the same in the results)?

Reply:
In SQLite, it's not the same for the "Group by week" use case:

- SQLite format: `YYYY-ww`
- Other databases format: `YYYY-MM-DD`

Let's sync in face-to-face this afternoon, coming to the Paris' office :)